### ISSUE TITLE
Insert template Icon ( perhaps dropdown menu)

### ISSUE THREAD

**Issue by dookei** [joplin/plugin-templates#70]
Hi there,
first of all many thanks for this plugin. I have been using it for some weeks now and is really helping me automate some tasks.
Maybe  one feature that I really miss is a shortcut icon for inserting templates in notes without to be always going throught the Tools-> Templates -> Insert Templates ->
![image](https://user-images.githubusercontent.com/24986481/205021901-0c8aa67d-14b7-4b70-b85b-da916625cabe.png)

Perhaps a drop down menu like when inserting table ?

**Comment by nishantwrp** [comment]
Yes, makes sense to have a insert template icon in the note toolbar.

**Comment by shinglyu** [comment]
Not sure if it's possible, but a "New note from template" icon next to the "New Note" icon would make a lot of sense:
![image](https://user-images.githubusercontent.com/3250983/221418416-262771d0-175f-49a1-8b02-95db4ab29d72.png)

**Comment by nishantwrp** [comment]
Yup, looks like it's not possible with the current api. https://joplinapp.org/api/references/plugin_api/enums/toolbarbuttonlocation.html

cc: @CalebJohn can you please confirm?

**Comment by CalebJohn** [comment]
It's not currently possible but would be a cool option to have. Although the next version of Joplin is moving the new note/new todo buttons, which will muddy the discussion. Either way, it's worth starting a discussion on the forum to add the capability.

**Comment by roman-r-m** [comment]
> It's not currently possible but would be a cool option to have. 

Hold my beer!
![ezgif com-video-to-gif](https://user-images.githubusercontent.com/995612/221695005-0ce14914-ff67-4f63-add8-0d97ae348a1a.gif)

All done from a plugin with zero changes to Joplin core. It's dirty hack though, so wouldn't recommend it.

**Comment by roman-r-m** [comment]
Here's a prototype: https://github.com/joplin/plugin-templates

**Comment by CalebJohn** [comment]
> Here's a prototype: https://github.com/joplin/plugin-templates/pull/74

Doing it because you can, not because your should :)
It looks nice though
